support-frontend/assets/pages/weekly-subscription-checkout/components/checkoutForm.jsx

```diff
@@ -0,0 +1,223 @@
+import Summary from 'components/subscriptionCheckouts/summary';
+import DirectDebitPopUpForm from 'components/directDebit/directDebitPopUpForm/directDebitPopUpForm';
+import type { PaymentAuthorisation } from 'helpers/paymentIntegrations/readerRevenueApis';
+import type { ErrorReason } from 'helpers/errorReasons';
+import type { ProductPrices } from 'helpers/productPrice/productPrices';
+import { titles } from 'helpers/user/details';
+import { formatMachineDate, formatUserDate } from 'helpers/dateConversions';
+import {
+  type FormActionCreators,
+  formActionCreators,
+  type FormField,
+  type FormFields,
+  getBillingAddress,
+  getDeliveryAddress,
+  getFormFields,
+  signOut,
+  type State,
+} from '../weeklySubscriptionCheckoutReducer';
+import { withStore } from 'components/subscriptionCheckouts/address/addressFields';
+import GridImage from 'components/gridImage/gridImage';
+import type { FormField as PersonalDetailsFormField } from 'components/subscriptionCheckouts/personalDetails';
+import PersonalDetails from 'components/subscriptionCheckouts/personalDetails';
+import { PaymentMethodSelector } from 'components/subscriptionCheckouts/paymentMethodSelector';
+import CancellationSection from 'components/subscriptionCheckouts/cancellationSection';
+import { newspaperCountries } from 'helpers/internationalisation/country';
+
+
+// ----- Types ----- //
+
+type PropTypes = {|
+  ...FormFields,
+  signOut: typeof signOut,
+  formErrors: FormError<FormField>[],
+  submissionError: ErrorReason | null,
+  productPrices: ProductPrices,
+  ...FormActionCreators,
+|};
+
+
+// ----- Map State/Props ----- //
+
+function mapStateToProps(state: State) {
+  return {
+    ...getFormFields(state),
+    formErrors: state.page.checkout.formErrors,
+    submissionError: state.page.checkout.submissionError,
+    productPrices: state.page.checkout.productPrices,
+  };
+}
+
+// ----- Form Fields ----- //
+
+const SelectWithLabel = compose(asControlled, withLabel)(Select);
+const FieldsetWithError = withError(Fieldset);
+
+const DeliveryAddress = withStore(newspaperCountries, 'delivery', getDeliveryAddress);
+const BillingAddress = withStore(newspaperCountries, 'billing', getBillingAddress);
+
+// ----- Component ----- //
+
+function CheckoutForm(props: PropTypes) {
+
+  return (
+    <Content modifierClasses={['your-details']}>
+      <Layout aside={(
+        <Summary
+          image={
+            <GridImage
+              gridId="checkoutPackshotPaperGraunVoucher"
+              srcSizes={[696, 500]}
+              sizes="(max-width: 740px) 50vw, 696"
+              imgType="png"
+              altText=""
+            />
+          }
+          title="weekly holding page"
+          description="placeholder description"
+          productPrice={{price: 99.89, currency: 'GBP'}}
+          promotion={undefined}
+          dataList={[
+          ]}
+          billingPeriod="Monthly"
+          changeSubscription={routes.paperSubscriptionProductChoices}
```

Comment:
This probably needs to change? But if this is just a placeholder at this stage then don't worry.